autoload/neomake/makers/ft/cpp.vim

```diff
@@ -9,12 +9,14 @@ endfunction
 function! neomake#makers#ft#cpp#clang() abort
     let maker = neomake#makers#ft#c#clang()
     let maker.exe = 'clang++'
+    let maker.args = makers.args + ['-std=c++1z']
     return maker
 endfunction
 
 function! neomake#makers#ft#cpp#gcc() abort
     let maker = neomake#makers#ft#c#gcc()
     let maker.exe = 'g++'
+    let maker.args = makers.args + ['-std=c++1z']
```

Comment:
I think my branch of the fork may have failed to load correctly into neovim or something because I didn't get any errors or warnings and that makes no sense. Sorry once more.